lib/model/content.dart

```diff
@@ -452,22 +452,24 @@ abstract class EmojiNode extends InlineContentNode {
 }
 
 class UnicodeEmojiNode extends EmojiNode {
-  const UnicodeEmojiNode({super.debugHtmlNode, required this.text});
+  const UnicodeEmojiNode({super.debugHtmlNode, required this.emojiName, required this.codepoints});
 
-  final String text;
+  final String emojiName;
+  final String codepoints;
```

Comment:
I think the name `emojiUnicode` that a previous revision had for this was a good one.

The name "codepoints" isn't great because it doesn't add much to the type `String` — it doesn't help much to identify which string this is, out of the various strings one might use to describe the emoji. In particular, my first guess from that name would actually be that it'd be the emoji code, like `12345-fedc-6789` or whatever, because that's an encoding of the list of codepoints.